services/git-service-impl/src/test/java/io/fabric8/launcher/service/github/GitHubServiceIT.java

```diff
@@ -57,4 +65,22 @@ protected String getRawFileUrl(String fullRepoName, String fileName) {
         return "https://raw.githubusercontent.com/" + fullRepoName + "/master/" + fileName;
     }
 
+    @Test
+    @Ignore
```

Comment:
Why @Ignored ? Might as well remove it?

Reply:
I could remove this one though. I used it to create a request and use it for deleteHook state